parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -259,6 +274,48 @@ public static Statistics toParquetStatistics(org.apache.parquet.column.statistic
     return stats;
   }
 
+  private static boolean shouldIgnoreStatistics(String createdBy) {
+    if (createdBy == null || !createdBy.startsWith("parquet-mr")) {
```

Comment:
Agreed. When a version of the writer can't be determined, this should consider the min/max stats invalid.